csv_template_ingester/tests/test_template_csv_parser.py

```diff
@@ -195,3 +195,12 @@ def test_create_list():
     assert len(lst) == 4
     assert lst[1] == '2'
 
+def test_add_fields_2():
```

Comment:
I don't love the naming convention of just adding _2.  Either combine this with the test_add_fields function or use a more specific name.

Reply:
I think it will make more sense to add this to the existing test_add_fields function. I will make the changes